examples/prometheus/src/main.rs

```diff
@@ -58,6 +58,7 @@ fn main() {
         .block_meta(Pipeline::new(BlockMetaParser, [Logger]))
         .metrics(vixen::metrics::Prometheus)
         .commitment_level(yellowstone_vixen::CommitmentLevel::Confirmed)
+        // .from_slot(slot_number)
```

Comment:
shouldn't it be in this way?
```suggestion
         .from_slot(None)
```

Reply:
Hey Sonic! Thanks for taking the time to review! No, in this particular one we are receiving the `u64` instead of the `Option<u64>` (because in general you would only call the method if you want to actually set the filter)